AzTS Data Table/Readme.md

```diff
@@ -0,0 +1,273 @@
+# AzTS Data Table
+
+
+## **Overview**
+
+Azure Tenant Security Solution centrally scans subscriptions and gets control scan results, processed RBAC data, resource inventory data, subscription metadata, etc., and pushes all the data into a blob storage of a central storage account.
+
+## **AzTS Data Table details**
+
+|Table Name|Description|
+|--|--|
+|[RBAC ](Readme.md#rbac)|Role-Based Access Control (RBAC) contains all PIM, permanent, and classic role assignment details.|
+|[Baseline controls](Readme.md#baseline-control)|Baseline controls contains a list of all controls (both baseline & non-baseline) supported by Azure Tenant Security (AzTS).|
+|[Control result](Readme.md#control-result)|Control result contains evaluated control scan result with appropriate status reason.|
+|[Resource Inventory](Readme.md#resource-inventory)|Resource Inventory contains details of all resources present in the subscription. It includes resource name, resource id, resource type, resource group name etc.|
+|[Subscription metadata](Readme.md#subscription-metadata)|Subscription metadata contains each subscription details like subscription id, tenant id, state, etc.|
```

Comment:
metadata "M" to be in caps. Pls. capitalize 1st letter of each.

Reply:
Done